test/integration/patients/worklist/worklist.js

```diff
@@ -4804,6 +4805,23 @@ context('worklist page', function() {
   });
 
   specify('flows with work:team:manage permission', function() {
+    const testCurrentClinician = getCurrentClinician({
+      relationships: {
+        role: getRelationship(roleTeamEmployee),
+        team: getRelationship(teamCoordinator),
+      },
+    });
+
+    const testNonTeamMemberClinician = getCurrentClinician({
```

Comment:
why use `getCurrentClinician` and not `getClinician`?  We might want to make `getCurrentClinician` error if `id` is passed as a key

Reply:
It's supposed to be `getClinician()`, my mistake. I updated it just now.